astroid/raw_building.py

```diff
@@ -467,18 +467,28 @@ def _astroid_bootstrapping():
     # Set the builtin module as parent for some builtins.
     nodes.Const._proxied = property(_set_proxied)
 
-    _GeneratorType = nodes.ClassDef(
-        types.GeneratorType.__name__, types.GeneratorType.__doc__
-    )
+    _GeneratorType = nodes.ClassDef(types.GeneratorType.__name__)
     _GeneratorType.parent = astroid_builtin
```

Comment:
Passing `parent` on L478 breaks a test as the `init` of `ClassDef` does something with `set_local` if `parent is not None`. That's why this is a separate line here.